apps/src/templates/projects/projectInfoDialog/ProjectInfoDialog.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useCallback} from 'react';
+import {useSelector} from 'react-redux';
+import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import Button from '@cdo/apps/templates/Button';
+import {
+  ProjectInfoDialogState,
+  hideProjectInfoDialog,
+} from './projectInfoDialogRedux';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import i18n from '@cdo/locale';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+const ProjectInfoDialog: React.FunctionComponent = () => {
+  const dispatch = useAppDispatch();
+  const isOpen = useSelector(
+    (state: {projectInfoDialog: ProjectInfoDialogState}) =>
+      state.projectInfoDialog.isOpen
+  );
+  const onClose = useCallback(
+    () => dispatch(hideProjectInfoDialog()),
+    [dispatch]
+  );
+
+  return (
+    <BaseDialog
+      isOpen={isOpen}
+      handleClose={onClose}
+      useUpdatedStyles
+      style={styles.dialog}
+    >
+      <h3>Project Info</h3>
```

Comment:
Don't we need to use DSCO Typography elements here for text?

Reply:
Yes - updated along with the i18n string. Thanks!